docs-beta/content/quickstart/_index.md

```diff
@@ -0,0 +1,466 @@
+---
+linktitle: "Quickstart"
+date: 2020-04-21T20:46:17-04:00
+weight: 3
+---
+
+<!-- TODO: Fix dates. Cross-platform date generation is a pain, so maybe use Docker locally? See what Netlify supports, or maybe there is a Hugo variable, or create a shortcode -->
+
+# Creating a Single Node M3DB Cluster
+
+This guide shows how to install and configure M3DB, create a single-node cluster, and read and write metrics to it.
+
+{{% notice warning %}}
+Deploying a single-node M3DB cluster is a great way to experiment with M3DB and get an idea of what it has to offer, but is not designed for production use. To run M3DB in clustered mode, with a separate M3Coordinator [read the clustered mode guide](cluster_hard_way.md).
+{{% /notice %}}
+
+## Prerequisites
+
+-   **Docker**: You don't need [Docker](https://www.docker.com/get-started) to run M3DB, but it is the simplest and quickest way.
+    -   If you use Docker Desktop, we recommend the following minimum _Resources_ settings.
+        -   _CPUs_: 2
+        -   _Memory_: 8GB
+        -   _Swap_: 1GB
+        -   _Disk image size_: 16GB
+-   **JQ**: This example uses [jq](https://stedolan.github.io/jq/) to format the output of API calls. It is not essential for using M3DB.
+-   **curl**: This example uses curl for communicating with M3DB endpoints. You can also use alternatives such as [Wget](https://www.gnu.org/software/wget/) and [HTTPie](https://httpie.org/).
+
+## Start Docker Container
+
+By default the official M3DB Docker image configures a single M3DB instance as one binary containing:
+
+-   An M3DB storage instance for time series storage. It includes an embedded tag-based metrics index and an etcd server for storing the cluster topology and runtime configuration.
+-   A coordinator instance for writing and querying tagged metrics, as well as managing cluster topology and runtime configuration.
+
+The Docker container exposes three ports:
+
+-   `7201` to manage the cluster topology, you make most API calls to this endpoint
+-   `7203` for Prometheus to scrape the metrics produced by M3DB and M3Coordinator
+-   `9003` to read and write metrics
```

Comment:
We honestly probably don't even need to expose this port since we'll just be telling users to interact with M3 through the coordinator.

Reply:
> We honestly probably don't even need to expose this port since we'll just be telling users to interact with M3 through the coordinator.

Good call, I checked through and everything works fine without it, so let's remove from the quickstart